functional-tests/src/test/groovy/FrontendMinespace/spec/PublicFrontendLogin.groovy

```diff
@@ -16,7 +16,7 @@ class  LoginPublicFrontEndPageSpec extends GebReportingSpec {
     def "I can log into the Minespace frontend given valid credentials"(){
         given:"I go to the homepage"
         to MinespaceFrontendLoginPage
-        LoginButton.click()
+        waitFor() { LoginButton.click() }
```

Comment:
The test seems to fail here more frequently than not, so im assuming it didnt fully wait for button to show up before clicking it